### DISCUSSION TITLE
full API with open source component.

### DISCUSSION THREAD

**Discussion by RDEV-K** [codewars/codewars.com#3288] [Q&A]
I have to implement API. I have experience in realizing API but this is some strange. The project is needed to implement full integration with this link- https://attack.mitre.org/resources/attack-data-and-tools
So there is no API doc and endpoint. so strange I don't know what I should do or where I start from.
If anyone has experience with this, give me some advise. I will be appreciate for that, thanks.

**Comment by hobovsky** [comment]
Codewars Github is not meant to discuss general programming questions. It's meant to discuss problems and ideas related to Codewars platform.

You might have better luck asking at some general programming forum, at some PHP forum, or at forum related to the component you want to use, its Discord, github, etc.

I am closing this discussion as off-topic.
